docs/quickstart/sample-minecraft.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Minecraft sample
+parent: Samples
+grand_parent: Quickstart
+nav_order: 2
```

Comment:
let's have this as "3" so it's listed below .NET Core sample and OpenArena

Reply:
Done.